libtransact/src/workload/runner.rs

```diff
@@ -389,9 +389,19 @@ fn slow_rate(
     submitted_batches: u32,
 ) -> Result<(), WorkloadRunnerError> {
     debug!("Received TooManyRequests message from target, attempting to resubmit batch");
-    // figure out the rate at which batches were successfully submitted
-    let effective_rate = submitted_batches / (time::Instant::now() - start_time).as_secs() as u32;
-    let wait = time::Duration::from_secs(1) / effective_rate;
+    let mut shutdown = false;
```

Comment:
This looks like it might be in the wrong commit.

Reply:
Moved to the correct commit